src/main/java/com/github/manolo8/darkbot/Bot.java

```diff
@@ -21,6 +24,7 @@ public static void main(String[] args) {
         } catch (UnsupportedLookAndFeelException e) {
             e.printStackTrace();
         }
+        STARTUP_PARAMS.parse(args);
```

Comment:
Startup params params = new StartupParams(args);
```suggestion
        StartupParams params = new StartupParams(args);
```